ocs/pkg/service/v0/users.go

```diff
@@ -103,7 +103,7 @@ func (o Ocs) GetUser(w http.ResponseWriter, r *http.Request) {
 	}
 
 	d := &data.User{
-		UserID:            account.PreferredName,
+		UserID:            account.Id,
```

Comment:
We already had the issue in the past, that this will break the test suite on dav paths. It expects the username there.

Reply:
Users that are created via ocs will have the account id provisioned the same as samaccontname and preferred username. the reva webdav api uses whatever is is the use.id.opaqueid, which for ocs provisioned accounts is the `userid`. That part should work.

But the sharing does not work because of https://github.com/cs3org/reva/pull/1281/files#diff-8d75e4b339ece6632172ba00f90e097b104b488bdeeee7e5b8576f04c9134480R198